ISM43362/ATParser/BufferedSpi/BufferedSpi.cpp

```diff
@@ -257,16 +258,18 @@ ssize_t BufferedSpi::read(uint32_t max)
     }
 
     enable_nss();
-    while (dataready.read() == 1 && (len < (_buf_size - 1))) {
+    while (dataready.read() == 1 && (len < (_buf_size - 2))) {
         tmp = SPI::write(0xAA);  // dummy write to receive 2 bytes
 
-        if (!((len == 0) && (tmp == 0x0A0D))) {
+        if (!((len == 0) && (tmp == 0x0A0D) && (FirstRemoved))) {
```

Comment:
Ok - could be useful to explain that DATA can be indeed starting with 0x0A0D  ... well done !!!